Gruntfile.js

```diff
@@ -31,18 +31,18 @@ module.exports = function( grunt ) {
 		// See https://github.com/jquery/sizzle/wiki/Sizzle-Documentation#browsers
 
 		browsers.desktop = [
-			"bs_chrome-43", "bs_chrome-44",
+			"bs_chrome-44", "bs_chrome-45",
 
 			"bs_firefox-31", "bs_firefox-38", // Firefox ESR
-			"bs_firefox-39", "bs_firefox-40",
+			"bs_firefox-40", "bs_firefox-41",
 
-			"bs_edge",
+			"bs_edge-12",
 
 			"bs_ie-9", "bs_ie-10", "bs_ie-11",
 
-			"bs_opera-30", "bs_opera-31",
+			"bs_opera-31", "bs_opera-32",
 
-			"bs_yandex-14.5",
+			"bs_yandex-14.5", "bs_yandex-14.12",
```

Comment:
Well, they've removed 14.5 so they don't seem interested to keep older
versions around (even Safari 7.0 was too much for them so Yandex will be
even less popular). But reach out if you think you can get anything out of
them.